docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md

```diff
@@ -81,54 +77,50 @@ mo --help
 
 ## Examples of CLI Commands
 
-Below is a list of separate examples for different frameworks and Model Optimizer parameters.
+Below is a list of separate examples for different frameworks and Model Optimizer parameters:
 
-1. Launch Model Optimizer for a TensorFlow MobileNet model in the binary protobuf format.
+- Launch Model Optimizer for a TensorFlow MobileNet model in the binary protobuf format:
```

Comment:
I guess it is a misprint to remove the number, but just to be sure I'm not missing something, a "comment" instead of a "suggestion" ;)

Reply:
My fault. I changed it to unordered list first and then I realized that there is a sub-list under this item. To avoid any formatting issues, I deleted my suggestion but somehow it's still committed..